setup.py

```diff
@@ -62,5 +63,6 @@
             [console_scripts]
             make_artifacts=vivarium_census_prl_synth_pop.tools.cli:make_artifacts
             make_results=vivarium_census_prl_synth_pop.tools.cli:make_results
+            build_results=vivarium_census_prl_synth_pop.tools.make_results:build_results
```

Comment:
I definitely think it's better to have build results as a runnable python script rather than as a click app. 

Also, click apps should all be defined in `cli.py`.

Reply:
As discussed, will try having `make_results -j` just turn around and run `make_results`. That way we can access both jobmony or normal processing w/o an additional cli tool